mucgpt-frontend/src/components/EdelweissSpinner/EdelweissSpinner.module.css

```
@@ -0,0 +1,50 @@
+.mark {
+    width: var(--edelweiss-spinner-size);
+    height: var(--edelweiss-spinner-size);
+    display: block;
+    flex: 0 0 auto;
+    transform-origin: center;
+    will-change: transform;
+    animation: edelweiss-spinner-spin 1500ms linear infinite;
+}
+
+.label {
+    color: currentColor;
```

Stylelint (17.11.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)